File: src/client_gen/ruby/my_strategy.rb
require_relative 'model'

class MyStrategy
    def get_action(player_view, debug_interface)
        raise NotImplementedError
    end
    def debug_update(player_view, debug_interface)
        debug_interface.send(DebugCommand::Clear.new())
        debug_interface.get_state()
    end
end

File: src/client_gen/csharp/MyStrategy.cs
using ProjectName.Model;

namespace ProjectName
{
    public class MyStrategy
    {
        public Action GetAction(PlayerView playerView, DebugInterface debugInterface)
        {
            throw new System.NotImplementedException();
        }

        public void DebugUpdate(PlayerView playerView, DebugInterface debugInterface)
        {
            debugInterface.Send(new DebugCommand.Clear());
            debugInterface.GetState();
        }
    }
}